CHANGELOG.md

```diff
@@ -2149,18 +2164,18 @@ rollback to 3.10.0 due to bug
 - Temporarily deactivated fauceting indication because it would activate when restoring an empty account.
 
 ## [1.3.2] - 2016-04-04
-
+### Uncategorized
 - When unlocking, first account is auto-selected.
 - When creating a first vault on the test-net, the first account is auto-funded.
 - Fixed some styling issues.
 
 ## [1.0.0] - 2016-03-25
-
-Made seed word restoring BIP44 compatible.
+### Uncategorized
+- Made seed word restoring BIP44 compatible.
```

Comment:
These older releases didn't have a `- ` in front of their changes. I've added it to make them consistent with the others.